src/ui/public/selectable_row/selectable_row.js

```diff
@@ -0,0 +1,22 @@
+import { uiModules } from 'ui/modules';
+import template from './selectable_row.html';
+
+const app = uiModules.get('kibana');
+
+app.directive('selectableRow', function () {
+  return {
+    restrict: 'E',
+    replace: true,
+    transclude: true,
+    template: template,
+    scope: {
+      id: '=',
+      isSelected: '=',
+      onSelectChange: '=',
+    },
+    controllerAs: 'selectableRow',
```

Comment:
I don't think we have an established convention around this in Kibana [at the moment](https://github.com/elastic/kibana/blob/master/style_guides/angular_style_guide.md).

I think the main pattern we are trying to avoid here is hanging properties directly of `$scope` in the controller JS code (e.g. `$scope.isSelected`) and referring to them with their bare names (e.g. `isSelected`) in the corresponding template. As long as that principle is followed, I'm good with either using `this` or a named controller via `controllerAs` in the template.

@BigFunger Do you have a strong opinion either way on this?